public/app/features/alerting/unified/components/contact-points/ContactPoints.v1.test.tsx

```diff
@@ -19,16 +17,19 @@ import {
 import { AccessControlAction, ContactPointsState } from 'app/types';
 
 import 'whatwg-fetch';
+import 'core-js/stable/structured-clone';
```

Comment:
This is a [jsdom limitation](https://github.com/jsdom/jsdom/issues/3363) at this moment.
I'd rather not include the polyfill where we use `structuredClone` because we don't need it there - browser support is decent, yet if you feel it's a better option I'm happy to change :)